### DISCUSSION TITLE
modified a rule list for suricata and can't remove the change

### DISCUSSION THREAD

**Discussion by jstreit15** [Security-Onion-Solutions/securityonion#9021] [General] [answered]
Well, actually i double and tripled down and now I can't remove the serveral changes I made to the rule.  When I do so-rule modify list this is what I see.
Modified rules + modifications:
  - 2008120 "\$HOME_NET" "[$HOME_NET, | $VONAGE]"
  - 2008120 "[$HOME_NET, | $VONAGE]" "[$HOME_NET, | $VONAGE]"
  - 2008120 "[$HOME_NET, | $VONAGE]" "[$HOME_NET, !$VONAGE]"
  - 2008120 "\$HOME_NET" "[$HOME_NET, !$VONAGE]"

and in a previous post I saw someone said to use this command but I can't figure out what to use from above to remove this command or a way to just go back to unedited version.  This was my first attempt at making a change and i won't loose anything if I recent everything.

so-rule modify remove <copy from so-rule modify list>

Thanks,

**Comment by dougburks** [comment, marked as answer]
One way to go back to the original version of the rule would be to do the following (replacing `<managername>_<role>` with your actual `managername` and `role`):
- make a backup copy of `/opt/so/saltstack/local/pillar/minions/<managername>_<role>.sls`
- edit `/opt/so/saltstack/local/pillar/minions/<managername>_<role>.sls` and remove your modifications from the `idstools` section

For more information, please see https://docs.securityonion.net/en/2.3/managing-alerts.html#modify-the-sid.

**Reply by jstreit15** [reply]
Thank you for this... I made the change.  

What is the proper command to run after I edit the .sls file?  I tried to run sudo so-rule-update but that didn't work.  I read the document you pointed to but could not seem to find it.  I ultimately ran the soup command to update it but that seems like a bit of overkill.

Thanks,

**Reply by dougburks** [reply]
As shown at https://docs.securityonion.net/en/2.3/managing-alerts.html#modify-the-sid:
```
sudo salt $SENSORNAME_$ROLE state.apply idstools
```
